local.go

```diff
@@ -35,7 +38,7 @@ func digest(URL string) (digest string, err error) {
 	if err != nil {
 		return
 	}
-	defer f.Close()
+	defer func() { _ = f.Close() }()
```

Comment:
gometalinter! This code passes all go linters possible, one of which checks that err is not handled